config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+
```

Comment:
you seem missing some routes relate to user in this file
be careful follow this chapter

Reply:
yeah i noticed it too. is it related with previous chapter? because previous chapter it showed that it did not pull request yet even though i already merged it